### PR TITLE
Created aside and styled

### LINKED ISSUE #3: Creating References Part
## 5. references

> assigned to Harun

__As a site visitor, I want to know how I can contact each of them__

### Repository

- [x] developed on a branch called `aside-text`

### HTML

- [X] An `aside` element ...
- [X] ... with references to our github profile
- [X] ... and class "references"

### CSS

- [X] the "references" class

### PR DESCRIPTION
**Pull request by harunaltunhr**
Hello, I created aside and styled. Could you please review it?

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.references{
```
Comment (Feruzteame) [inline style.css:2]: hey harun it looks great but i think it is better to add `.references{border: solid 3px black}`. we have to work depend on the wire-frame.

**Review by Feruzteame — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (Feruzteame) [reply style.css:1]: And it needs margin to make space b/n the boxes.

Comment (harunaltunhr) [reply style.css:2]: A border came from .profile and .references will have same border, but we can change also its color and style. But wireframe do not order us to do a border. It only shows us the position of elements? Am I wrong?

**Review by harunaltunhr — COMMENTED** [review]
How it looks?

Aside

**Review by adekimpianna — APPROVED** [review]
Thanks harun for the changes! I think it looks great now

**Review by miroslavveljanoski — APPROVED** [review]
This looks nice.

**Pull request merged**